docs/src/__examples__/Accordion/ACTIONS.tsx

```diff
@@ -0,0 +1,126 @@
+        <SeatButton direction={direction} currentSeat={`${rowNumber}D`} />
+      </Stack>
+    );
+    const SeatMap = ({ direction }) => (
+      <Stack>
+        <Stack direction="row" align="center">
+          <Stack inline grow={false} align="center" direction="column">
+            <Text>A</Text>
+            <SeatButton direction={direction} currentSeat="1A" />
+          </Stack>
+          <Stack inline grow={false} align="center" direction="column">
+            <Text>B</Text>
+            <SeatButton direction={direction} currentSeat="1B" />
+          </Stack>
+          <Stack inline grow={false} align="center" direction="column">
+            <Text> </Text>
+            <Text> </Text>
+            <Text>1</Text>
+          </Stack>
+          <Stack inline grow={false} align="center" direction="column">
+            <Text>C</Text>
+            <SeatButton direction={direction} currentSeat="1C" />
+          </Stack>
+          <Stack inline grow={false} align="center" direction="column">
+            <Text>D</Text>
+            <SeatButton direction={direction} currentSeat="1D" />
+          </Stack>
+        </Stack>
+        <SeatRow direction={direction} rowNumber={2} />
+        <SeatRow direction={direction} rowNumber={3} />
+        <SeatRow direction={direction} rowNumber={4} />
+        <SeatRow direction={direction} rowNumber={5} />
+        <SeatRow direction={direction} rowNumber={6} />
+        <SeatRow direction={direction} rowNumber={7} />
+      </Stack>
+    );
+
+    return (
+      <Accordion expandedSection={expandedSection} onExpand={id => setExpandedSection(String(id))}>
+        <AccordionSection
+          actions={
+            <Button type="secondary" onClick={() => setExpandedSection("outbound")}>
+              Show seats to Boston
+            </Button>
+          }
+          id="outbound"
+          header={
+            <Heading as="h3" type="title3">
+              Seating for Barcelona
+              <Icons.FlightDirect ariaLabel=" to " />
+              Boston
+            </Heading>
+          }
+        >
+          <SeatMap direction="outbound" />
+        </AccordionSection>
+        <AccordionSection
+          actions={
+            <Button type="secondary" onClick={() => setExpandedSection("inbound")}>
+              Show seats to Barcelona
+            </Button>
+          }
+          id="inbound"
+          header={
+            <Heading as="h3" type="title3">
+              Seating for Boston
+              <Icons.FlightDirect ariaLabel=" to " />
+              Barcelona
+            </Heading>
+          }
+        >
+          <SeatMap direction="inbound" />
+        </AccordionSection>
+      </Accordion>
+    );
+  },
+  info: {
+    title: "Section actions",
+    description:
+      "To override the open button and add your own actions in the section heading, use the <code>actions</code> property of the section.",
+  },
+};
```

Comment:
Is there a reason to be including these in the file but not the example?

Reply:
it was moved from current examples, did not include it by myself.